CONTRIBUTING.md

```diff
@@ -0,0 +1,102 @@
+# Contributing Guide
+
+👍🎉 First off, thanks for taking the time to contribute! 🎉👍
+
+Filing an Issue or a Pull Request is often the best way to address a problem with this library;
+however, we may not get to these right away. Although we try to be quick, our primary, daily focus is
+writing code. If you want a timely response (especially if you have an emergency), we recommend
+you contact our [offcial support team](https://support.recurly.com/).
+
+#### Table Of Contents
+
+* [I don't want to read this whole thing, I just have a question!!!](#i-dont-want-to-read-this-whole-thing-i-just-have-a-question)
+* [I think something is wrong with this library](#i-think-something-is-wrong-with-this-library)
+* [I know what's wrong and I want to submit a code change](#i-know-whats-wrong-and-i-want-to-submit-a-code-change)
+  * [Development Dependencies](#development-dependencies)
+  * [Building and Testing](#building-and-testing)
+  * [Formatting Code](#formatting-code)
+  * [Generated Code](#generated-code)
+
+
+## I don't want to read this whole thing I just have a question!!!
+
+The best way to get a question answered is through our [offcial support channel](https://support.recurly.com/). If you
+have a specific question related to this library and cannot find an answer, feel free to post an issue with the question.
+
+## I think something is wrong with this library
+
+Are you getting an exception or seeing some unexpected behavior from the library? An issue is the way to go.
+Submit an issue and make sure you provide as much detail as possible. Some things you'll want to include:
+
+* Your dependency versions (client version, language version, OS, etc)
+* A stack trace if available
+* A [Minimal, Reproducible Example](https://stackoverflow.com/help/minimal-reproducible-example)
+
+The more information you give us, the quicker and easier the response will be. Keep in mind that issues should be scoped
+to a problem in this library and we may often redirect you to our official support for problems originating from
+upstream systems.
+
+## I know what's wrong and I want to submit a code change
+
+So, you are sure you want to submit a change to the code? We appreciate the help!
+Before you do, I'd consider opening a discussion in the form of an issue. This accomplish a few things:
```

Comment:
The whole document is written in 2nd person except this sentence. Recommend changing to:
```
Before you do, consider opening a discussion in the form of an issue
```